.ci/docker/ci_commit_pins/pytorch.txt

```diff
@@ -1 +1 @@
-release/2.3
+release/2.3.1
```

Comment:
You should not change this. It refers to the branch name, and there is not a branch called releaes/2.3.x